1.1/jessie/runtime/Dockerfile

```diff
@@ -6,8 +6,8 @@ ENV DOTNET_HOSTING_OPTIMIZATION_CACHE /packagescache
 COPY ./build-cache.sh /packagescache/build-cache.sh
 
 # set up package cache
-RUN chmod +x /packagescache/build-cache.sh \
-    && /packagescache/build-cache.sh https://dist.asp.net/packagecache/1.1.0/aspnetcore.packagecache-1.1.0-legacy-debian.8-x64.tar.gz \
+RUN chmod +x /packagescache/build-cache.sh
+RUN /packagescache/build-cache.sh https://dist.asp.net/packagecache/1.1.0/aspnetcore.packagecache-1.1.0-legacy-debian.8-x64.tar.gz \
```

Comment:
Yeah, the issue Nate linked is the reason. Presumably we don't have to chmod inside the container at all, and I had intended to look at removing it completely if it was possible.